### PR TITLE
Code Documentation

### PR DESCRIPTION
**Pull request by Naylin15**
Hi, @hectorerb 

This PR in progress adds the documentation to all the Swift sources.

### CODE REVIEW

File: FlyveMDMInventoryTests/BatteryTest.swift [hunk 1]
```diff
@@ -23,6 +23,7 @@ class BatteryTest: XCTestCase {
         super.tearDown()
     }
 
+    /// Get the state of the battery
```
Comment (ajsb85) [inline FlyveMDMInventoryTests/BatteryTest.swift]: 3 slashes?

Comment (hectorerb) [reply FlyveMDMInventoryTests/BatteryTest.swift]: @ajsb85 in swift you may use /** ... */ for multi-line or /// for single-line comments

Comment (Naylin15) [reply FlyveMDMInventoryTests/BatteryTest.swift]: Yes, Hector explained to me that it is with 3 slashes the single line documentation so Jazzy can read the comment properly

**Comment by Naylin15** [comment]
Hi, @hectorerb

This code documentation is finished

Could you please help me with the reviews?

Thanks

**Pull request merged**